### PR TITLE
Feature/home/action

### PR DESCRIPTION
**Pull request by at-hanhpham**
- action select nearItemCell
- select item in category cell

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: Meete/Controllers/Search/SearchViewController.swift [hunk 1]
```diff
@@ -51,63 +54,75 @@ class SearchViewController: BaseViewController {
...
+            }
+        }
+    }
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        updateUI()
+    }
+
+    private func fakeDataItems() {
```
Comment (at-daiho) [inline Meete/Controllers/Search/SearchViewController.swift]: Add Todo dummy data

File: Meete/Controllers/Search/SearchViewController.swift [hunk 2]
```diff
@@ -123,6 +138,25 @@ class SearchViewController: BaseViewController {
...
+            if !isDisplayedMap {
+            } else {
+                hideMap()
+                isDisplayedMap = false
+            }
+        } else {
+            if !isDisplayedMap {
+                showMap()
+                isDisplayedMap = true
+            } else {
```
Comment (at-daiho) [inline Meete/Controllers/Search/SearchViewController.swift]: What is this at here?

File: Meete/Controllers/Search/SearchViewController.swift [hunk 3]
```diff
@@ -123,6 +138,25 @@ class SearchViewController: BaseViewController {
         styleFilterButton.setAttribute(titleLineBreakMode: .byWordWrapping, numberOfLine: 2, borderWidth: 0, imageButton: #imageLiteral(resourceName: "icFilter"), state: .normal, tintColor: Color.lightRed)
     }
 
+    private func updateUI() {
+        if !searchSetting.doWantShowMap {
+            if !isDisplayedMap {
```
Comment (at-daiho) [inline Meete/Controllers/Search/SearchViewController.swift]: What is this at here?

File: Meete/Controllers/Search/SearchViewController.swift [hunk 4]
```diff
@@ -150,7 +184,12 @@ class SearchViewController: BaseViewController {
         navigationItem.leftBarButtonItem = searchButton
     }
 
-    // MARK: - Action Of PickerView
+    // MARK: - Action
+    @IBAction private func selectFillter(_ sender: UIButton) {
+        buttonSelectedIndex = sender.tag
```
Comment (at-daiho) [inline Meete/Controllers/Search/SearchViewController.swift]: buttonSelectedIndex is not clearly, please rename

File: Meete/Controllers/Search/SearchViewController.swift [hunk 5]
```diff
@@ -332,31 +364,31 @@ extension SearchViewController: UISearchBarDelegate {
...
         switch buttonName {
         case .category:
-            stringFilter = listCategoryFilter[row].name
+            stringFilter = categories[row].name
         case .area:
-            stringFilter = listAreaFilter[row].name
+            stringFilter = areas[row].name
         case .filter:
-            stringFilter = listStyleFilter[row].name
+            stringFilter = sortStyles[row].name
```
Comment (at-daiho) [inline Meete/Controllers/Search/SearchViewController.swift]: sortStyles -> sortTypes?

File: Meete/Controllers/Search/SearchViewController.swift (hunk 2 above)
Comment (at-hanhpham) [inline Meete/Controllers/Search/SearchViewController.swift]: I want check map hide or show.
if doWantShowMap = true but map are showing then do nothing, if map are hiding then show Map.

File: Meete/Controllers/Home/HomeViewController.swift [hunk 6]
```diff
@@ -80,13 +96,23 @@ extension HomeViewController: UITableViewDelegate {
...
     func newItemCell(_ newItemCell: NewItemCell, didSelect indexPath: IndexPath) {
         let detailVC = DetailViewController()
         navigationController?.pushViewController(detailVC, animated: true)
     }
 }
 
+// MARK: - CategoryCellDelegate
+extension HomeViewController: CategoryCellDelegate {
+    func categoryCell(_ categoryCell: CategoryCell, didSelectItem index: Int) {
+        // dummy data
```
Comment (at-daiho) [inline Meete/Controllers/Home/HomeViewController.swift]: Add TODO

**Pull request closed without merging**